app/views/nonprofit_applications/index.html.slim

```diff
@@ -24,11 +24,11 @@
     h2.npo-application-subheader Application
     - if !@interest_form
       p.npo-application-notify.static-page-desc
-          span style="font-weight: 700; display: block" You must fill out the Interest Form from the previous step first.
+          span style="font-weight: 700; display: block" You must fill out the interest form and complete a quick informational phone call first!
     - elsif !@proposal_open
       p.npo-application-notify.static-page-desc
         span style="font-weight: 700; display: block" This part of the application is not available to you just yet
-        '  We will allow you to submit an application after completing a short phone call with us. If we don't contact you within a couple days,
+        '  We will allow you to submit an application after completing a short phone call with us. If you have questions,
```

Comment:
I think friendlier language would be "We'll open the second part of the application after you complete..." ? Instead of "allow"

Reply:
ahhh yes great catch LOL "we will allow" is so weird